File: 1684. Count the Number of Consistent Strings-统计一致字符串的数目/1684. Count the Number of Consistent Strings-统计一致字符串的数目.c
int countConsistentStrings(char * allowed, char ** words, int wordsSize){
    int res = 0;
    for(int i=0;i<wordsSize;i++){
        bool flag = true;
        for(int j = 0,ilen = strlen(words[i]); j < ilen; j++){
            if(!strchr(allowed,words[i][j])){
                flag = false;
                break;
            }
        }
        if(flag){
            res++;
        }
    }
    return res;
}

File: 1684. Count the Number of Consistent Strings-统计一致字符串的数目/1684. Count the Number of Consistent Strings-统计一致字符串的数目.js
/**
 * @param {string} allowed
 * @param {string[]} words
 * @return {number}
 */
var countConsistentStrings = function(allowed, words) {
    let res = 0;
    for(let i = 0, len = words.length; i < len; i++){
        let flag = true
        for(let j = 0, ilen = words[i].length; j < ilen; j++){
            if(!allowed.includes(words[i][j])){
                flag = false
                break
            }
        }
        if(flag){
            res++
        }
    }
    return res
};
